controllers/controller.js

```diff
@@ -52,3 +52,25 @@ exports.updateVotes = (req, res, next) => {
         })
 }
 
+exports.getArticles = (req, res, next) => {
+
+    const topic = req.query.topic;
+
+    if (Object.keys(req.query).length > 0) {
+        selectArticlesByQuery(topic)
```

Comment:
Implement this logic in your SQL query, using conditional WHERE clauses, as covered [in the notes here](https://notes.northcoders.com/courses/js-back-end/complex-queries); much cleaner way to do it